@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -218,28 +230,28 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
         {...rest}
         {...omit(inputProps, ["aria-invalid"])}
         ref={mergedRefs}
+        type="text"
+        role="combobox"
         value={value}
-        onBlur={() => virtualFocus.moveFocusToTop()}
+        onBlur={composeEventHandlers(onBlur, virtualFocus.moveFocusToTop)}
```

Comment:
I needed `onBlur` for the example, so I only added `composeEventHandlers` on that one for now. But should we add it to the other event handlers as well?

Reply:
Doesn't hurt to add it to all of them i guess, but not needed in this PR 🤷